### PR TITLE
Router pon: ADD IliadBox, AVM FRITZ!Box 5530/5590, ZTE F6645P

### PR DESCRIPTION
**Pull request by simonebortolin**

### CODE REVIEW

File: _router_pon/ont-zte-f6645p.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+| dev  | size     | erasesize | name             |
+| ---- | -------- | --------- | ---------------- |
+| mtd0 | 01000000 | 00010000  | "whole flash"    |
+| mtd1 | 00080000 | 00010000  | "uboot"          |
+| mtd2 | 00700000 | 00010000  | "kernel0"        |
+| mtd3 | 00700000 | 00010000  | "kernel1"        |
+| mtd4 | 00010000 | 00010000  | "others"         |
+| mtd5 | 00010000 | 00010000  | "parameter tags" |
+| mtd6 | 00160000 | 00010000  | "usercfg"        |
```
Comment (ErnyTech) [inline _router_pon/ont-zte-f6645p.md:40]: No

```
dev:    size   erasesize  name
mtd0: 08000000 00020000 "whole flash"
mtd1: 00200000 00020000 "u-boot"
mtd2: 00400000 00020000 "others"
mtd3: 00400000 00020000 "parameter tags"
mtd4: 00400000 00020000 "wlan"
mtd5: 00800000 00020000 "usercfg"
mtd6: 00400000 00020000 "Plugin"
mtd7: 02a00000 00020000 "kernel1"
mtd8: 02a00000 00020000 "kernel2"
mtd9: 029e0000 00020000 "rootfs"
```

File: _router_pon/ont-zte-f6645p.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+|              |                                                                   |
+| ------------ | ----------------------------------------------------------------- |
+| Vendor/Brand | ZTE                                                               |
+| Model        | F6645P                                                            |
+| ODM          | ✅                                                                |
+| CPU          | ZTE                                                               |
+| CPU Clock    | 266 MHz                                                           |
+| Chipset      | ZTE                                                               |
+| Flash        |                                                                   |
+| RAM          |                                                                   |
```
Comment (ErnyTech) [inline _router_pon/ont-zte-f6645p.md:19]: Flash: Kioxia TC58BVG0S3HTAI0 130 MB
RAM: 315 MB

File: _router_pon/ont-zte-f6645p.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+| System       |                                                                   |
+| 2.5GBaseT    | Yes                                                               |
+| Optics       | SC/APC or SC/UPC                                                  |
+| IP address   | 192.168.1.1                                                       |
+| Web Gui      | ✅ user `admin`, password `admin` or user `user`, password `user` |
+| SSH          |                                                                   |
+| Telnet       | ✅ [^1]                                                           |
+| Serial       | ✅                                                                |
```
Comment (ErnyTech) [inline _router_pon/ont-zte-f6645p.md:27]: Only RX

File: _router_pon/ont-zte-f6645p.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+| CPU          | ZTE                                                               |
+| CPU Clock    | 266 MHz                                                           |
+| Chipset      | ZTE                                                               |
+| Flash        |                                                                   |
+| RAM          |                                                                   |
+| System       |                                                                   |
+| 2.5GBaseT    | Yes                                                               |
+| Optics       | SC/APC or SC/UPC                                                  |
+| IP address   | 192.168.1.1                                                       |
+| Web Gui      | ✅ user `admin`, password `admin` or user `user`, password `user` |
```
Comment (ErnyTech) [inline _router_pon/ont-zte-f6645p.md:24]: Random password

File: _router_pon/ont-zte-f6645p.md [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+syn_version
+```
+
+# Use
+
+## Enable Telnet
+{% include alert.html content="This is an external script ([ZTE Telnet enabler](https://github.com/douniwan5788/zte_modem_tools)), use at your own risk! Credentials don't survive at reboot!" alert="Note"  icon="svg-info" color="blue" %}
+
+{% include alert.html content="For italian users, the script above only works on versions V6.0.10N40 (TIM) and V6.0.10P6N7 (OpenFiber)" alert="Note"  icon="svg-info" color="blue" %}
+
```
Comment (ErnyTech) [inline _router_pon/ont-zte-f6645p.md:65]:
```suggestion
```

**Review by ErnyTech — DISMISSED** [review]

**Review by nanomad — APPROVED** [review]

**Pull request merged**